jsk_perception/node_scripts/people_pose_estimation_2d.py

```diff
@@ -490,6 +503,8 @@ def _pose_estimate_chainer_backend(self, bgr_img):
                 bgr_img, joint_cands_indices, all_peaks, candidate)
         else:
             result_img = bgr_img
+        if not (self.width is None or self.height is None):
+            result_img = cv2.resize(result_img, (org_w, org_h))
 
         return result_img, self._extract_joint_position(joint_cands_indices, candidate)
```

Comment:
@iory `self._extract_joint_position` is not affected by resizing the input for the NN?